src/config.ts

```diff
@@ -0,0 +1,17 @@
+import type { WebSocketConfig } from './types';
+
+const serverDefault: WebSocketConfig = {
+  connectTimeoutTime: 120,
+  keepAliveIntervalTime: 1_000,
+  keepAliveTimeoutTime: 10_000,
+  streamBufferSize: 1024 * 1024, // 1MB
+};
+
+const clientDefault: WebSocketConfig = {
+  connectTimeoutTime: Infinity,
+  keepAliveIntervalTime: 1_000,
+  keepAliveTimeoutTime: 10_000,
+  streamBufferSize: 1024 * 1024, // 1MB
+};
```

Comment:
For libraries the default behaviour should be no keep alive.

Reply:
Same with the `connectTimeoutTime`.

Reply:
Changed to:
```ts
const serverDefault: WebSocketConfig = {
  connectTimeoutTime: Infinity,
  keepAliveIntervalTime: Infinity,
  keepAliveTimeoutTime: Infinity,
  streamBufferSize: 1 * 1024 * 1024, // 1MB
};

const clientDefault: WebSocketConfig = {
  connectTimeoutTime: Infinity,
  keepAliveIntervalTime: Infinity,
  keepAliveTimeoutTime: Infinity,
  streamBufferSize: 1 * 1024 * 1024, // 1MB
};
```